pkgloader-recommended.sty

```diff
@@ -172,3 +172,17 @@
 %    \end{macrocode}
 %
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+
+
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+%
+%  lstingsutf8
+%
+%    \begin{macrocode}
+\Load {listingsutf8} after  {listings}
+                 if loaded
+                 because {listingsutf8 patches macros of
+                          listings}
```

Comment:
No need to break this line here. These files don't go into the documentation .pdf. Also, please align the lower lines with the `after` keyword and remove the additional space after `after`.

Reply:
The line break was because of consistency for the other rules. I'll remove them, because I have a wide screen with more than 80 chars space.